sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -723,6 +723,19 @@ class ExplainSuiteAE extends ExplainSuiteHelper with EnableAdaptiveExecutionSuit
       assert(inMemoryRelationNodeId != columnarToRowNodeId)
     }
   }
+
+  test("SPARK-38232: Explain formatted does not collect subqueries under query stage in AQE") {
```

Comment:
Thank you for adding this at `ExplainSuiteAE`. Do we have a similar test coverage in `ExplainSuite` or somewhere-else already, @ulysses-you ?

Reply:
yes, we have good test for it in `SQLQueryTestSuite` include aqe and subquery, see `explain.sql` and `explainaqe.sql`. But unfortunately, the test of `explainaqe.sql` only test the init plan.

Reply:
Thank you for confirming.